src/libraries/System.IO.Pipelines/Directory.Build.props

```diff
@@ -2,5 +2,6 @@
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
     <StrongNameKeyId>Open</StrongNameKeyId>
+    <IsNETCoreApp>true</IsNETCoreApp>
```

Comment:
I understand why we're making this inbox, but was people aware that we needed to make this inbox to support `System.Net.Connections`? 

You also need to update https://github.com/dotnet/runtime/blob/a366d638d880823ad63d28c22ddee0aeafbbcfc1/src/libraries/pkg/baseline/packageIndex.json#L3347-L3356

To add `"net5.0"` to `"InboxOn"` object.

Reply:
> I understand why we're making this inbox, but was people aware that we needed to make this inbox to support `System.Net.Connections`?

Yep, discussed in and out of API review